src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -70,6 +70,7 @@
         + " -j,--javadocTree            Print Parse tree of the Javadoc comment%n"
         + " -J,--treeWithJavadoc        Print full Abstract Syntax Tree of the file%n"
         + " -o <arg>                    Sets the output file. Defaults to stdout%n"
+        + " -omitIgnoredModules <arg>   Sets ability to not run modules that are ignored.%n"
```

Comment:
I am leaning towards `executeIgnoredModules`. @romani Are you ok with this?

I don't see a reason for short form right now.